packaging/nnstreamer.spec

```diff
@@ -232,6 +245,7 @@ Tizen internal API to construct the pipeline without the permissions.
 cp %{SOURCE1001} .
 %if %{with tizen}
 cp %{SOURCE1002} .
+cp %{SOURCE1002} ./nnstreamer-cpp.manifest
```

Comment:
outside the cond %if %{with tizen}

Reply:
or, how about removing capi-nnstreamer.manifest file and copy SOURCE1001(nnstreamer.manifest) with named capi-nnstreamer.manifest

Reply:
I've moved it out of the cond. Refactoring manifest and with-tizen conditions may be done later with another PR.